src/commands/deployment/helper.ts

```diff
@@ -0,0 +1,221 @@
+import Table from "cli-table";
+import * as fs from "fs";
+import * as os from "os";
+import Deployment from "spektate/lib/Deployment";
+import AzureDevOpsPipeline from "spektate/lib/pipeline/AzureDevOpsPipeline";
+import IPipeline from "spektate/lib/pipeline/Pipeline";
+import { logger } from "../../logger";
+
+let hldPipeline: IPipeline;
+let clusterPipeline: IPipeline;
+let srcPipeline: IPipeline;
+const fileLocation = os.homedir() + "/.Spektate";
+export let config: { [id: string]: string } = {};
+export enum OUTPUT_FORMAT {
+  NORMAL = 0,
+  WIDE = 1,
+  JSON = 2
+}
+
+export class Helper {
+  public static initializePipelines() {
+    srcPipeline = new AzureDevOpsPipeline(
+      config.AZURE_ORG,
+      config.AZURE_PROJECT,
+      false,
+      config.AZURE_PIPELINE_ACCESS_TOKEN
+    );
+    hldPipeline = new AzureDevOpsPipeline(
+      config.AZURE_ORG,
+      config.AZURE_PROJECT,
+      true,
+      config.AZURE_PIPELINE_ACCESS_TOKEN
+    );
+    clusterPipeline = new AzureDevOpsPipeline(
+      config.AZURE_ORG,
+      config.AZURE_PROJECT,
+      false,
+      config.AZURE_PIPELINE_ACCESS_TOKEN
+    );
+  }
+
+  public static verifyAppConfiguration = (callback?: () => void) => {
+    if (
+      config.STORAGE_TABLE_NAME === "" ||
+      config.STORAGE_TABLE_NAME === undefined ||
+      config.STORAGE_PARTITION_KEY === "" ||
+      config.STORAGE_PARTITION_KEY === undefined ||
+      config.STORAGE_ACCOUNT_NAME === "" ||
+      config.STORAGE_ACCOUNT_NAME === undefined ||
+      config.STORAGE_ACCOUNT_KEY === "" ||
+      config.STORAGE_ACCOUNT_KEY === undefined ||
+      config.GITHUB_MANIFEST_USERNAME === "" ||
+      config.GITHUB_MANIFEST_USERNAME === undefined ||
+      config.MANIFEST === "" ||
+      config.MANIFEST === undefined ||
+      config.AZURE_PROJECT === "" ||
+      config.AZURE_PROJECT === undefined ||
+      config.AZURE_ORG === "" ||
+      config.AZURE_ORG === undefined
+    ) {
+      Helper.configureAppFromFile(callback);
+    } else {
+      Helper.initializePipelines();
+      if (callback) {
+        callback();
+      }
+    }
+  };
+
+  public static configureAppFromFile = (callback?: () => void) => {
+    fs.readFile(fileLocation, (error, data) => {
```

Comment:
If we're expecting the file to contain text, we should use `fs.readFile(fileLocation, 'utf8', (error, data)...`. This will automatically make `data` a string